daily.sh

```diff
@@ -133,7 +133,7 @@ check_dependencies() {
     local ver_71=$(php -r "echo (int)version_compare(PHP_VERSION, '7.1.3', '<');")
     local ver_72=$(php -r "echo (int)version_compare(PHP_VERSION, '7.2.5', '<');")
     local python3=$(python3 -c "import sys;print(int(sys.version_info < (3, 4)))" 2> /dev/null)
-    local python_deps=$(scripts/check_requirements.py > /dev/null 2>&1; echo $?)
+    local python_deps=$("${LIBRENMS_DIR}/scripts/check_requirements.py" > /dev/null 2>&1; echo $?)
```

Comment:
This looks weird.
Didn't test but maybe this?
```suggestion
    local python_deps=$("${LIBRENMS_DIR}/scripts/check_requirements.py > /dev/null 2>&1; echo $?")
```

Reply:
That is what I did first :)
It makes it attempt to run the whole thing as a command including the redirects.